code/game/machinery/computer/communications.dm

```diff
@@ -95,8 +95,8 @@
 		if ("answerMessage")
 			if (!authenticated(usr))
 				return
-			var/answer_index = text2num(params["answer"])
-			var/message_index = text2num(params["message"])
+			var/answer_index = params["answer"]
+			var/message_index = params["message"]
```

Comment:
You'll still want to verify these are numbers in the case of malicious user input.

Reply:
I've added checks against either malicious or otherwise erroneous user input, in that case.